Use the canonical pattern files to generate prometheus configurations

stephen-soltesz opened the pull request:
To normalize the configurations used by scraper and prometheus, this PR updates `generate_prometheus_targets.sh` to use the standard patterns files for what defines a testing, staging, or production machine.


<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/m-lab/operator/122)
<!-- Reviewable:end -->

pboothe commented on generate_prometheus_targets.sh:
```diff
@@ -6,15 +6,13 @@ for project in mlab-sandbox mlab-staging mlab-oti ; do
 
 # All testing sites and machines.
-SELECT_mlab_sandbox=".*[a-z]{3}[0-9]t.*"
+SELECT_mlab_sandbox=$( cat plsync/testing_patterns.txt )
```
No `| xargs | sed -e 's/ /|.*/g' ` ?

stephen-soltesz commented:
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[generate_prometheus_targets.sh, line 9 at r1](https://reviewable.io:443/reviews/m-lab/operator/122#-KnAQ8-eblGR83k13C8I-r1-9:-KnASmrgl_edclOPZAOs:b-srcxlt) ([raw file](https://github.com/m-lab/operator/blob/7391e415240930780b5dbeff081a4ee154c1af19/generate_prometheus_targets.sh#L9)):*
<details><summary><i>Previously, pboothe (Peter Boothe) wrote…</i></summary><blockquote>

No `| xargs | sed -e 's/ /|.*/g' ` ?
</blockquote></details>

Good idea. Also, I didn't realize until now that the `.*` patterns are unnecessary.  So, I've removed that from the sed expressions. And, updated the npad pattern to include all subpatterns.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/operator/122)*
<!-- Sent from Reviewable.io -->

stephen-soltesz commented:
PTAL?

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/operator/122#-:-KnAYL1WZJOqnmChGlAk:b-833305)*
<!-- Sent from Reviewable.io -->
